pkg/pusher/pusher.go

```diff
@@ -147,16 +148,36 @@ LOOP:
 				// for now ignoring the receipt and checking only for error
 				_, err = s.pushSyncer.PushChunkToClosest(ctx, ch)
 				if err != nil {
-					if !errors.Is(err, topology.ErrNotFound) {
-						logger.Debugf("pusher: error while sending chunk or receiving receipt: %v", err)
+					if errors.Is(err, topology.ErrWantSelf) {
+						// we are the closest ones - this is fine. set the chunk as synced
+						// this is to make sure that the sent number does not diverge from the synced counter
+						// the edge case is on the uploader node, in the case where the uploader node is
+						// connected to other nodes, but is the closest one to the chunk.
+						t, err = s.tag.Get(ch.TagID())
```

Comment:
just  set  a variable to true and conditionally increment sent count on line 176